DiceKitTests/AdditionExpression_Tests.swift

```diff
@@ -118,3 +118,51 @@ extension AdditionExpression_Tests {
     }
     
 }
+
+// MARK: - Operators
+// TODO: Make these SwiftChecks
+extension AdditionExpression_Tests {
+    
+    func test_operator_shouldWorkWithIntAndDie() {
+        property("Constant + Die returns correct AdditionExpression") <- forAll {
+            (a: Int, b: Int) in
+            
+            let constant = c(a)
+            let die = d(b)
+        
+            let expression1 = constant + die
+            let expression2 = die + constant
+            
+            let test1 = (expression1 == AdditionExpression(constant, die))
```

Comment:
do `let expectedExpression1 = ...` in the arrange section, then do the compare here.